upgrade/helpers/constants/environment_config.py

```diff
@@ -0,0 +1,90 @@
+import os
+
+from upgrade.helpers.constants import settings
+
+
+FROM_VERSION = settings.UPGRADE.FROM_VERSION
+TO_VERSION = settings.UPGRADE.TO_VERSION
+
+OS = settings.UPGRADE.OS
+
+if OS == "rhel6":
+    CAPSULE_TOOLS_REPO = settings.REPOS.SATTOOLS_REPO.RHEL6
+    RHEV_CAPSULE_AK = settings.UPGRADE.CAPSULE_AK.CAPSULE_AK_RHEL6
+    TOOLS_URL_RHEL = settings.REPOS.SATTOOLS_REPO.RHEL6
+elif OS == "rhel7":
+    CAPSULE_TOOLS_REPO = settings.REPOS.SATTOOLS_REPO.RHEL7
+    RHEV_CAPSULE_AK = settings.UPGRADE.CAPSULE_AK.CAPSULE_AK_RHEL7
+    TOOLS_URL_RHEL = settings.REPOS.SATTOOLS_REPO.RHEL7
+elif OS == "rhel8":
+    CAPSULE_TOOLS_REPO = settings.REPOS.SATTOOLS_REPO.RHEL8
+    RHEV_CAPSULE_AK = settings.UPGRADE.CAPSULE_AK.CAPSULE_AK_RHEL8
+    TOOLS_URL_RHEL = settings.REPOS.SATTOOLS_REPO.RHEL8
```

Comment:
We really dont need this file at all since formatting of settings (like CAPSULE_TOOLS_REPO here) as per another setting can be done at YAML level itself using jinja/YAML formatting.

Refer https://github.com/SatelliteQE/robottelo/blob/master/conf/repos.yaml.template

Reply:
Checked https://github.com/SatelliteQE/satellite6-upgrade/pull/467/files#r602082977